services/minespace-web/src/components/pages/Project/InformationRequirementsTablePage.js

```diff
@@ -126,31 +152,34 @@ const StepForms = (props, state, next, prev, close, handleTabChange, handleIRTSu
       >
         Back
       </Button>,
-      <Popconfirm
-        placement="top"
-        title="Are you sure you want to cancel the submission for this IRT?"
-        okText="Yes"
-        cancelText="No"
-        onConfirm={() => {
-          this.close();
-        }}
-        disabled={state.submitting}
-      >
-        <Button type="secondary" className="full-mobile" disabled={state.submitting}>
-          Cancel
-        </Button>
-      </Popconfirm>,
-      <Button
-        type="primary"
-        style={{ display: "inline", float: "right" }}
-        htmlType="submit"
-        onClick={(event) => {
-          handleIRTSubmit(event);
-        }}
-        disabled={state.submitting}
-      >
-        Submit IRT
-      </Button>,
+      <>
+        {props.project?.information_requirements_table?.status_code === "REC" ? (
+          <Link
+            to={routes.REVIEW_INFORMATION_REQUIREMENTS_TABLE.dynamicRoute(
+              props.project?.project_guid
+            )}
+          >
+            <Button
+              type="primary"
+              style={{ display: "inline", float: "right" }}
+              htmlType="submit"
+              onClick={() => handleIRTUpdate({ status_code: "UNR" }, "IRT submitted ")}
+              disabled={state.submitting}
+            >
+              Submit IRT
+            </Button>
+          </Link>
+        ) : (
+          <Button
+            type="primary"
+            style={{ display: "inline", float: "right" }}
+            htmlType="submit"
+            disabled={state.submitting}
```

Comment:
does this button also need the handleIRTUpdate() on click?

Reply:
To be removed. Need to work on Update in future work.